app/ci_program.py

```diff
@@ -121,7 +121,7 @@ def commandLoop(self):
               # The sequence was empty, so it looks like this Escape wasn't
               # really the start of a sequence and is instead a stand-alone
               # Escape. Just forward the esc.
-              ch = curses.ascii.ESC
+              ch = app.curses_util.KEY_ESCAPE
```

Comment:
Oh, yeah. I guess it would work either way here; but if we optimize for integers then an int would be better for that reason. cool.